mobile/src/Student/screens/Game/PhaseOneBasicGamePlay/index.js

```diff
@@ -72,38 +72,51 @@ const PhaseOneBasicGamePlay = ({
     }, [gameSession, currentTime])
 
     const handleSubmitAnswer = () => {
-
```

Comment:
I think this might be an older version of PhaseOneBasicGamePlay.  I think Josephina's previous updates removed the alert window and provided some behavior on the button that seems to be being reverted here. Would you be able to check that your synched up with dev for this?